centreon/www/front_src/src/ResourceAccessManagement/translatedLabels.ts

```diff
@@ -0,0 +1,6 @@
+export const labelResourceAccessRules = 'Resource access rules';
+export const labelRules = 'Name';
+export const labelDescription = 'Description';
+export const labelActions = 'Actions';
+export const labelStatus = 'Status';
+export const labelSearch = 'Search';
```

Comment:
you may need to add the translation in .po file

Reply:
Done